app/javascript/packs/dashboardDropdowns.js

```diff
@@ -0,0 +1,126 @@
+import { initializeDropdown } from '@utilities/dropdownUtils';
+
+function getFormValues(form) {
+  const articleId = form.action.match(/\/(\d+)$/)[1];
+  const inputs = form.getElementsByTagName('input');
+  const formData = { id: articleId, article: {} };
+
+  for (let i = 0; i < inputs.length; i += 1) {
```

Comment:
Wow, that `i += 1` really upset the C programmer in me, I did not expect that. 😂  Is use of `i++` not common in JS anymore?

Reply:
I'm not used to seeing it either 😆  I'll change it to save our eyes